cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -28,6 +28,8 @@ spec:
     # Whether each active MDS instance will have an active standby with a warm metadata cache for faster failover.
     # If false, standbys will be available, but will not have a warm cache.
     activeStandby: true
```

Comment:
If activeStandbyCount is specified, `activeStandby` will be ignored, right? So in this example let's remove `activeStandby` so we can be clear that it is the preferred setting.

Reply:
ack